node_modules/oae-mixpanel/lib/api.content.js

```diff
@@ -0,0 +1,124 @@
+/*!
+ * Copyright 2015 Apereo Foundation (AF) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+var _ = require('underscore');
+var Validator = require('oae-util/lib/validator').Validator;
+
+var MixPanelUtil = require('./util');
+
+/**
+ * Get the total size of uploaded files for the current month for all tenants
+ * or the current tenant, depending on the used router.
+ * @see https://mixpanel.com/docs/api-documentation/data-export-api
+ *
+ *  * Exporter return example:
+ *  *
+ *  *   {
+ *  *       status: "ok",
+ *  *       results: {
+ *  *           2015-07-29: 2489,
+ *  *           2015-07-30: 2989176,
+ *  *           2015-07-31: 670234
+ *  *       }
+ *  *   }
+ *
+ * @param  {Context}    ctx                 Standard context object containing the current user and the current tenant
+ * @param  {Function}   callback            Standard callback function
+ * @param  {Object}     callback.err        An error that occurred, if any
+ * @param  {Object}     callback.stats      Object containing the stats for the tenant(s)
+ */
+var getUploadedFilesFileSize = module.exports.getUploadedFilesFileSize = function(ctx, callback) {
+    var mixPanelConfig = require('../../../config').config.mixpanel;
```

Comment:
The application configuration is not obtained in this way, because a different config file location can be used.

Instead, you should hook into the initialize sequence with an `init.js` and initialize this module with the configuration with which the application is being initialized. See other `init.js` files for reference.